app/javascript/components/MemberDetailsForm/MemberDetailsForm.js

```diff
@@ -56,22 +56,17 @@ export class MemberDetailsForm extends Component {
     this.prefill();
   }
 
-  getPrefillValue(name) {
-    if (!this.props.prefillValues) return null;
-
-    return this.props.prefillValues[name];
-  }
-
   prefill() {
     const data = {};
     for (const field of this.props.fields) {
-      data[field.name] =
-        this.getPrefillValue(field.name) || field.default_value;
+      data[field.name] = this.props.formValues[field.name]
```

Comment:
Did you mean `this.props.formValues ? ... : ... ` here?

Reply:
ouch! good catch.

Reply:
Actually, it's already there! I was confused. The ternary operator expands to multiple lines, check the following 2 lines.

Reply:
Yeah, but the first line tries to access a property (but the object could be undefined).

Reply:
Although I'm not sure if it's intended or just being overprotective? `formValues` is always an object in the reducer.

Reply:
Ideally the store should make sure that formValues is always an object, but right now it's not doing that. I'll add the check.